docs/client-development.md

```diff
@@ -1,81 +0,0 @@
-Notices For Client Library Development
```

Comment:
I agree that client-develpment.md should be removed in pegasus repo, but I suggest adding this document in client repo or official website.

Reply:
Certainly. But we don't have to wait after this doc added to website to merge this PR. I will do it later this day.